CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Changes
 
+## Version 2.2.0
+* Improve batch-id (UUID) performance #109
```

Comment:
Should this be #109 (PR) or #108 (issue)?

Reply:
fixed - use the issue id